### ISSUE TITLE
Solution on public CTF

### ISSUE THREAD

**Issue by SakiiR** [ChrisTheCoolHut/PinCTF#5]
Hey ! 

Just to say, I think you are using an exemple which is from the Root-Me website. I think this is forbidden to share solution (Flag here, pretty bad).

Would be cool if you can just use a different flag to display :+1:

**Comment by ChrisTheCoolHut** [comment]
You're right! I had no idea this was a reverseable from root-me. I'll be on the hunt for a new binary that solves backwards.

**Comment by SakiiR** [comment]
:)

I don't know if it will help you, but here is something I do which is wery similar: https://github.com/SakiiR/timeauth

Maybe the service given as example can be used with your tool :)
